app/models/user.rb

```diff
@@ -21,6 +22,23 @@ def self.from_omniauth(auth_hash)
     # roles = auth_hash['extra']['raw_info']['resource_access']['account']['roles']
   end
 
+  def token_invalid?
```

Comment:
Could we add `argo` to this? Since the user will have more than one kind of token. 
```suggestion
  def argo_token_invalid?
```

Reply:
done ✔️